djoser/logging.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework_tracking.mixins import BaseLoggingMixin
+
+import inspect
+
+def mixin(obj):
+    # Add BaseLogingMixin from drf-tracking in first sequence of mixin to the class
+    bases = obj.__bases__
+    obj.__bases__ = (BaseLoggingMixin , )
+    obj.__bases__ += bases
```

Comment:
@fichie23 
`obj.__bases__ = (BaseLoggingMixin, obj.__bases__)`?

Reply:
It will be tuple inside tuple @orhan89  , but yeah it can be simplified to be like this `obj.__bases__ = (BaseLoggingMixin , ) + obj.__bases__`